doc/source/ray-air/examples/training_ingest_object_classification.ipynb

```diff
@@ -0,0 +1,1232 @@
+      "Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in /home/ray/anaconda3/lib/python3.9/site-packages (from jsonschema->ray[data]) (0.19.3)\n",
+      "Requirement already satisfied: charset-normalizer<4,>=2 in /home/ray/anaconda3/lib/python3.9/site-packages (from requests->ray[data]) (2.0.4)\n",
+      "Requirement already satisfied: idna<4,>=2.5 in /home/ray/anaconda3/lib/python3.9/site-packages (from requests->ray[data]) (3.4)\n",
+      "Requirement already satisfied: certifi>=2017.4.17 in /home/ray/anaconda3/lib/python3.9/site-packages (from requests->ray[data]) (2023.5.7)\n",
+      "Requirement already satisfied: six>=1.5 in /home/ray/anaconda3/lib/python3.9/site-packages (from python-dateutil<3.0.0,>=2.1->botocore==1.31.18->awscli) (1.13.0)\n",
+      "\u001b[33mWARNING: Ignoring invalid distribution -otocore (/home/ray/anaconda3/lib/python3.9/site-packages)\u001b[0m\u001b[33m\n",
+      "\u001b[0m"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install \"ray[data]\" torchvision awscli"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's first download an image dataset to use. If you already have an image dataset on local disk or in S3, you can skip the rest of this section.\n",
+    "Otherwise, follow on to download a dataset from Kaggle.\n",
+    "For this example, we'll use bash to download a [4GB subset of the ImageNet dataset](https://www.kaggle.com/datasets/ifigotin/imagenetmini-1000) and (optionally) upload it to an S3 bucket.\n",
+    "\n",
+    "```bash\n",
+    "$ pip install kaggle awscli\n",
+    "$ kaggle datasets list -s ifigotin/imagenetmini-1000\n",
+    "# ref                         title                 size  lastUpdated          downloadCount  voteCount  usabilityRating  \n",
+    "# --------------------------  --------------------  ----  -------------------  -------------  ---------  ---------------  \n",
+    "# ifigotin/imagenetmini-1000  ImageNet 1000 (mini)   4GB  2020-03-10 01:05:11          11779        133  0.375\n",
+    "$ kaggle datasets download ifigotin/imagenetmini-1000\n",
+    "$ unzip imagenetmini-1000.zip -d /tmp\n",
+    "```\n",
+    "\n",
+    "The directory structure should look like this:\n",
+    "```bash\n",
+    "$ tree /tmp/imagenet-mini\n",
+    "# ── train\n",
+    "# │   ├── n01440764\n",
+    "# │   │   ├── n01440764_10043.JPEG\n",
+    "# │   │   ├── n01440764_10470.JPEG\n",
+    "# │   │   ├── n01440764_10744.JPEG\n",
+    "# ...\n",
+    "# └── val\n",
+    "#     ├── n01440764\n",
+    "#     │   ├── ILSVRC2012_val_00009111.JPEG\n",
+    "#     │   ├── ILSVRC2012_val_00030740.JPEG\n",
+    "# ...\n",
+    "```\n",
+    "\n",
+    "If you want to read from S3, run the following commands and pass the S3 URI `s3://imagenetmini-1000/train` in the following Python code.\n",
+    "Otherwise, you can pass the local path `/tmp/imagenet-mini/train`.\n",
```

Comment:
```suggestion
    "If you want to read from S3, run the following commands and pass the S3 URI `s3://imagenetmini-1000/train` in the following Python code.\n",
    "Otherwise, pass the local path `/tmp/imagenet-mini/train`.\n",
```